corehq/apps/accounting/invoicing.py

```diff
@@ -347,12 +347,13 @@ def __init__(self, date_start, date_end, account, recipients=None):
         self.subscriptions = {}
 
     def create_invoice(self):
-        for subscription in self.account.subscription_set.all():
-            if should_create_invoice(subscription, subscription.subscriber.domain, self.date_start, self.date_end):
-                if subscription.plan_version in self.subscriptions:
-                    self.subscriptions[subscription.plan_version].append(subscription)
+        for sub in self.account.subscription_set.all():
```

Comment:
nit: adding a filter to subscription_set would avoid adding complexity to the if statement.